Cargo.toml

```diff
@@ -332,6 +330,7 @@ internal_benches = []
 slow_tests = []
 std = ["alloc"]
 test_logging = ["std"]
+boringssl_no_prefix = []
```

Comment:
Let's avoid using the name "boringssl". Instead let's name it something like `no_versioned_extern`